src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,46 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final String SPACE = " ";
+    private static final String DASH = " - ";
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate localDateFrom = LocalDate.parse(dateFrom,
+                DateTimeFormatter.ofPattern("dd.MM.yyyy"));
```

Comment:
I'd just recommend creating a constant for this datetimeformatter as described in common mistakes